### PR TITLE
Pal 1082 remove resource parent

### PR DESCRIPTION
**Pull request by dev930018**
Awaiting on [PAL-1023](https://github.com/gchq/Palisade-clients/pull/67) being merged.

### CODE REVIEW

**Review by pd104923 — DISMISSED** [review]
1 minor thing and the license changes should be reverted.

File: client-java/src/unit-tests/java/uk/gov/gchq/palisade/client/java/internal/download/DownloaderExceptionTest.java [hunk 1]
```diff
@@ -29,8 +29,8 @@ void testGetStatusCode() {
         var expectedCode = 400;
         var exception = new DownloaderException("oops", expectedCode);
         assertThat(exception.getStatusCode())
-            .as("check %s's status code", DownloaderException.class.getSimpleName())
-            .isEqualTo(400);
+                .as("check %s's status code", DownloaderException.class.getSimpleName())
+                .isEqualTo(400);
```
Comment (developer01189998819991197253) [inline client-java/src/unit-tests/java/uk/gov/gchq/palisade/client/java/internal/download/DownloaderExceptionTest.java:33]:
```suggestion
                .isEqualTo(expectedCode);
```

File: licenses/gnu_lgpl_2.1.html [hunk 2]
```diff
@@ -29,6 +29,30 @@
 <!-- start of server/banner.html -->
 <!-- start of head-include-2.html -->
 <meta name="viewport" content="width=device-width, initial-scale=1" />
+
```
Comment (developer01189998819991197253) [inline licenses/gnu_lgpl_2.1.html:32]: revert

File: licenses/mit_license.html [hunk 3]
```diff
@@ -18,7 +18,7 @@
 <link type="text/css" rel="stylesheet" href="https://opensource.org/files/css/css_7rptTjxTlnnwcQOT_3B6dmWJfzo0pnNKsCvuv3HArrk.css" media="all" />
 
 <!--[if (lt IE 9)]>
-<link type="text/css" rel="stylesheet" href="https://opensource.org/sites/all/themes/bootstrap-business/css/ie8.css?qqdhfs" media="all" />
```
Comment (developer01189998819991197253) [inline licenses/mit_license.html:21]: revert

**Review by developer01189998819991197253 — CHANGES_REQUESTED** [review]

**Review by developer01189998819991197253 — APPROVED** [review]

**Pull request merged**